docs/usage.rst

```diff
@@ -2,9 +2,49 @@
 Usage
 =====
 
+Quickstart
+~~~~~~~~~~
+
 To use figanos in a project::
 
     import figanos.matplotlib as fg
     fg.utils.set_mpl_style('ouranos')
 
 The style can be applied to any matplotlib figures, even if they are not created with figanos.
+
+Installing Logos
```

Comment:
I would change the order of this section.  Ouranos first, general case second. The vast majority of our users (all right now) are Ouranos. It needs to be really clear that they need to install the logos when they install figanos.